projects/storefrontlib/src/shared/components/list-navigation/pagination/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './pagination.builder';
 export * from './pagination.component';
 export * from './pagination.model';
 export * from './pagination.module';
+export * from './testing/pagination-testing.module';
```

Comment:
`pagination-testing.module` is private testing util not to be exposed in public API yet (it's incomplete solution). Please import from absolute path in tests, if needed.